pokemon-checker/src/components/PokemonDisplay/PokemonDisplay.tsx

```diff
@@ -49,7 +49,7 @@ class PokemonDisplay extends React.Component<displayProps, displayState> {
   fetchPokemonObject() {
     const url = this.props.pokemonUrl;
     if (url && url !== "") {
-      getSelectedPokemon(url)
+      this.props.getPokemonData(url)
```

Comment:
Because of how much context this function needs, I've moved it to be passed in as a prop at the App level from the PokemonService.